src/content/blog/windows-ddev-setup.md

```diff
@@ -0,0 +1,58 @@
+
+Two recent Windows machines I set up were the new ARM64/Qualcomm/CoPilot variety. They were excellent and fast and had great battery life. There's very little I had to do differently with them, but I'll mention in the steps when there was something different. (I was surprised by the lack of a fingerprint sensor on both, but the Windows Hello facial recognition was quite fast. There is no ARM64 Discord app, and the AMD64 one had horrific performance.)
+
+1. Remove unwanted applications. I usually start by removing a bunch of bloatware. 
+2. Do all Windows updates. 
+3. Turn off the `System` -> `Notifications` -> `Additional settings` that cause the “Windows experience” prompts after upgrade. 
+4. Enable Windows Update->Advanced Options->Receive updates for other Microsoft Products. Amazingly, this is not on by default, and you might have an old WSL2 kernel! (See [Beware of Dirty Pipes](beware-of-dirty-pipes-and-docker-desktop-on-windows.md).)
+5. Install important apps. I always start with these. Each of these except Discord had an ARM64 version. If on an ARM64 machine, make sure you get the right version!
+    -  Notion
+    -  Chrome
+    -  1Password
+    -  PhpStorm (Although there are many ways to use PhpStorm on WSL2, I just open the project in `\\wsl.localhost\Ubuntu\home\rfay\workspace\<project>` and it works great and performance is fine.)
+    -  GoLand
+    -  Discord
+    -  Slack
+    - [Perforce P4V](https://www.perforce.com/downloads/helix-visual-client-p4v) is my favorite merge conflict resolver.
+6. In PowerShell, `wsl --install` and `wsl --update`
+7. Windows Terminal is a fantastic terminal and is installed by default these days. I always set it up early with "Default Terminal Application: Windows Terminal" and "Interaction->Automatically Copy Selection to Clipboard", and set Ubuntu as default, and have it auto-start on login. 
+8. Once Ubuntu is installed:
+    -  `sudo apt update && sudo apt install -y apt-transport-https autojump build-essential ca-certificates ccache clang curl dirmngr etckeeper expect git gnupg jq libcurl4-gnutls-dev libnss3-tools lsb-release mariadb-client nagios-plugins net-tools postgresql-client unzip vim xdg-utils zip`
+    -  sudo snap install —classic go
+    -  sudo snap install ngrok and ngrok config add-authtoken
+9. In Windows Explorer, add my WSL2 home directory to favorites by copying it into the favorites area.
+10. Run the [DDEV WSL2 install script](https://ddev.readthedocs.io/en/stable/users/install/ddev-installation/#wsl2-docker-ce-inside-install-script).
+    * On ARM64 this will fail the Chocolatey installation because Chocolatey doesn't plan to support ARM64. But you can ignore the failure; the script continues on anyway.
+    * On ARM64, consider `choco uninstall -y mkcert gsudo` so that DDEV can get the native versions of each of these.
```

Comment:
just based on the instructions i dont understand that point. in line 39 you say chocolatey doesnt support ARM64 and then you write in 40 that the user should run choco uninstall mkcert?  based on line 39 i thought chocolatey doesnt work with arm64 and line 40 is only applying a chocolatey command on arm64?! this is confusing just based on the instructions?

Reply:
```suggestion
    * On ARM64, `choco uninstall -y mkcert gsudo` so that the DDEV installer can get the native versions of each of these.
```

Reply:
Choco installs amd64 versions in most cases. The DDEV installer doesn't allow that.